deployer/deployer-services/src/main/java/io/hyscale/deployer/services/util/DeployerLogUtil.java

```diff
@@ -50,12 +50,12 @@
 	@Autowired
 	private LogProcessor logProcessor;
 
-	public void processLogs(DeploymentContext context) throws HyscaleException {
+	public void processLogs(DeploymentContext context, String podName) throws HyscaleException {
 
 		if (context.isTailLogs()) {
-			tailLogs(context);
+			tailLogs(context, podName);
```

Comment:
during logs also deployment context is only used for deploy only. Others should just read the parameters.